hack/goarch.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
```

Comment:
posix SH for maximum portability, the idea is to just call docker ASAP.

we'll need something else for windows.